homework/tests/homework.spec.js

```diff
@@ -0,0 +1,230 @@
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('4');
+      await page.locator('#number2Field').type('k');
+      await page.selectOption('#selectOperationDropdown', {label: 'Subtract'});
+      await page.locator('#calculateButton').click();
+  
+      var error = await page.textContent('#errorMsgField');
+      await expect(error).toContain('Number 2 is not a number');
+    });
+
+    test('Multiplying 4 by k results in an error message', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('4');
+      await page.locator('#number2Field').type('k');
+      await page.selectOption('#selectOperationDropdown', {label: 'Multiply'});
+      await page.locator('#calculateButton').click();
+  
+      let error = await page.textContent('#errorMsgField');
+      await expect(error).toContain('Number 2 is not a number');
+    });
+
+    test('Dividing 4 from k results in an error message', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('4');
+      await page.locator('#number2Field').type('k');
+      await page.selectOption('#selectOperationDropdown', {label: 'Divide'});
+      await page.locator('#calculateButton').click();
+  
+      let error = await page.textContent('#errorMsgField');
+      await expect(error).toContain('Number 2 is not a number');
+    });
+
+    test('Adding 2 and 2 results in 4 and press clear button and then it should clear the field', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('2');
+      await page.locator('#number2Field').type('2');
+      await page.selectOption('#selectOperationDropdown', {label: 'Add'});
+      await page.locator('#calculateButton').click();
+      await page.locator('#clearButton').click();
+
+      var answer = await page.textContent('#numberAnswerField');
+      expect(answer).toMatch('');
+    });
+
+    test('Integers only checkbox test', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('3.7');
+      await page.locator('#number2Field').type('2.5');
+      await page.selectOption('#selectOperationDropdown', {label: 'Add'});
+      await page.locator('#calculateButton').click();
+
+      let answer = await page.inputValue('#numberAnswerField');
+      console.log(answer);
```

Comment:
Console logging here is redundant. 
Please don't leave code which was meant for debugging in a pull request.